webapp/app/forms/steps/lotse/lotse_step.py

```diff
@@ -7,6 +7,7 @@
 class LotseFormSteuerlotseStep(FormSteuerlotseStep):
     template = 'basis/form_standard.html'
     header_title = None
+    disable_extended_footer = True
```

Comment:
Why not set this for the SteuerlotseStep with False and then set it to True here?

Reply:
Done